.travis.yml

```diff
@@ -26,12 +23,10 @@ deploy:
     overwrite: true
     on:
       tags: true
-  # TODO: Use 'travis encrypt --com -r adafruit/<repo slug>' to generate
-  #       the encrypted password for adafruit-travis. Paste result below.
   - provider: pypi
     user: adafruit-travis
     password:
-      secure: #-- PASTE ENCRYPTED PASSWORD HERE --#
+      secure: gFlUJx4AFvVTzhbRbTiBXl3be3Nm+m9VHCSA6ICatUdO34NA7Wzo+rBoV7YDjV85Pc3U9G5zW9SvXb9BMIT7k4tOqmog5RCtzwlLq3S/RRoOkGr6UY702EC1M0ruMVHBBAB5Pwzfom8VVRNL9n4nXsPfr41U35M6daIYH+onEDXIhq+ES3NkU3ciTC/q9QO9ChsHuWS1oNjGxmiYWqDEJ6vYL34D2RYIq4+P/hmFFM9rvhYBRYT+MYeHBOXbsc5KZhoIki7rKU0UW4zQcORkvMuWfalLWd24GR8aG9rB9fZrnAZG1qGtqwBCh91CNlzt8eziPOGeP1hXMnVHnvXveumnSBrFcGvkQ2DYu3v6kUn9AVMoJ3oeDpegCzse7jgOLUipJ0eXN1lmEKpEZNlszRGlSA2m2mYVo4WwqMSz7ZtZWG7L+4OQ21JG+MbACoQJJ//YxAwFtALQrVlOaE9X7ElCDnlceYRebm7RTQ0bE4+BTL08ppKyOaiFJxdbSz3ZPufNHVlM8oYqIwRPLLZozMG5WmtZ62EO3Jvb0YicuccO2xNCHW0zySxQgbzVMfLnN4F9wSxcLIADbXnvPx1A5D6zd1CxFK16S49CJZm0QUPlXMA7ac1rwKv6sZhPA4/rsoOLbkPTaqZZtrwmc4/+9qRxSDbR0YDAmR5ZMnQjT5I=
```

Comment:
I'm not certain this is tabbed correctly. I checked a couple other PRs for PyPI and the line starts next to `secure:`. It may be an artifact of the line being longer and GitHub formatting it this way. If it fails, this is the first place you should look.